src/apps/frontend/pages/authentication/login/login-form.tsx

```diff
@@ -20,91 +22,95 @@ const LoginForm: React.FC<LoginFormProps> = ({ onError, onSuccess }) => {
-                name="password"
-                onChange={formik.handleChange}
-                onBlur={formik.handleBlur}
-                placeholder="Enter your password"
-                type={isPasswordVisible ? 'text' : 'password'}
-                value={formik.values.password}
-              />
+              name="password"
+              onChange={formik.handleChange}
+              onBlur={formik.handleBlur}
+              placeholder="Enter your password"
+              type={isPasswordVisible ? 'text' : 'password'}
+              value={formik.values.password}
+            />
 
-              <button
-                className="absolute inset-y-0 right-0 flex items-center px-4"
-                onClick={(e) => {
-                  e.preventDefault();
-                  togglePasswordVisibility();
-                }}
-              >
-                {isPasswordVisible ? (
-                  <img
-                    className="size-6.5 opacity-65"
-                    src="/assets/img/icon/eye-closed.svg"
-                    alt="hide password icon"
-                  />
-                ) : (
-                  <img
-                    className="size-6.5 opacity-65"
-                    src="/assets/img/icon/eye-open.svg"
-                    alt="show password icon"
-                  />
-                )}
-              </button>
-            </FormControl>
-          </div>
+            <button
+              className="absolute inset-y-0 right-0 flex items-center px-4"
+              onClick={(e) => {
+                e.preventDefault();
+                togglePasswordVisibility();
+              }}
+            >
+              {isPasswordVisible ? (
+                <img
+                  className="size-6.5 opacity-65"
+                  src="/assets/img/icon/eye-closed.svg"
+                  alt="hide password icon"
+                />
+              ) : (
+                <img
+                  className="size-6.5 opacity-65"
+                  src="/assets/img/icon/eye-open.svg"
+                  alt="show password icon"
+                />
+              )}
+            </button>
+          </FormControl>
+        </div>
 
-          <button
-            className={`w-full cursor-pointer rounded-lg border border-primary bg-primary p-4 font-medium text-white transition hover:bg-primary/90
-              ${isLoginLoading && 'cursor-not-allowed bg-primary/90'}`
-            }
-            disabled={isLoginLoading}
-            type="submit"
-          >
-            Sign In
-          </button>
-        </form>
-      </div>
-    </div>
+        <button
+          className={`w-full cursor-pointer rounded-lg border border-primary bg-primary p-4 font-medium text-white transition hover:bg-primary/90
+            ${isLoginLoading && 'cursor-not-allowed bg-primary/70'}`
+          }
+          disabled={isLoginLoading}
+          type="submit"
+        >
+          {isLoginLoading ? 'Loading...' : 'Sign In'}
```

Comment:
Have we checked Tailwind in general? What is their treatment when for async operation?

This may give some idea - https://tailwindcss.com/docs/hover-focus-and-other-states